internal/interface/client/pendo/pendo.go

```diff
@@ -0,0 +1,12 @@
+package pendo
+
+type PendoLabels map[string]string
+
+type PendoMetrics struct {
+	VisitorID string            `json:"visitorId"`
+	Values    map[string]string `json:"values"`
+}
```

Comment:
Fine as a placeholder, while we work out the actual shape of the data we must submit :)